packages/camera/src/components/Controls/FullScreenButton.js

```diff
@@ -1,10 +1,9 @@
+import PropTypes from 'prop-types';
 import React from 'react';
-import { useTranslation } from 'react-i18next';
 import { Text } from 'react-native';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
 
-export default function FullScreenButton() {
-  const { t } = useTranslation();
-
+export default function FullScreenButton({ label }) {
```

Comment:
instead of a `label` prop i think it's better to use a boolean prop called `isInFullScreenMode`